app/components/live-sidebar.js

```diff
@@ -3,14 +3,21 @@ import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 import { ROLES } from '../constants/live';
 import { getOwner } from '@ember/application';
+import { inject as service } from '@ember/service';
 
 export default class LiveHeaderComponent extends Component {
+  @service live;
```

Comment:
same as above if live service is not used then remove this one too.

Reply:
fixeed